README.md

```diff
@@ -154,9 +155,8 @@ Examples:
 =====
 Make sure to take a look at the examples, which show the library in action. Let me know if you have questions, or other issues.
 
-
-Used Libraries:
+Editing the Project
 =====
- This version uses the public domain licensed CocoaAsyncSocket library 
- for UDP-socket networking.
- [Click here](https://github.com/robbiehanson/CocoaAsyncSocket) for more information.
+To edit the project it would be convenirnt to open `SwiftCoAP.xcworkspace` in Xcode. This way it'll open both Client and Server example implementations targets and the Package. You may also want to edit the library code separately. In such a case just open `Package.swift` with Xcode.
```

Comment:
```suggestion
To edit the project it would be convenient to open `SwiftCoAP.xcworkspace` in Xcode. This way it'll open both Client and Server example implementations targets and the Package. You may also want to edit the library code separately. In such a case just open `Package.swift` with Xcode.
```

Reply:
Why not apply the typos :D